app/models/profile.rb

```diff
@@ -7,6 +7,12 @@ class Profile < ActiveRecord::Base
   scope :active, -> { where(active: true) }
   scope :verified, -> { where.not(verified_at: nil) }
 
+  enum deactivation_reason: {
+    not_deactivated: 0,
```

Comment:
double negative?

maybe `active` instead?

or could we allow `deactivation_reason` to be `NULL`?